javascript-if/main.js

```diff
@@ -0,0 +1,86 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  let result;
+  if (number < 5) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isEven(number) {
+  let result;
+  if (number % 2 === 0) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function startsWithJ(string) {
+  let result;
+  if (string.charAt(0) === 'J') {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrink(person) {
+  let result;
+  if (person.age < 21) {
+    result = false;
+  } else {
+    result = true;
+  }
+  return result;
+}
+
+function isOldEnoughToDrive(person) {
+  return false;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age < 999) {
+    result = false;
+  }
+  return result;
```

Comment:
this function should just return false.

Reply:
Oops I updated the wrong function... I have a change another one.